pkg/dumper/dumper.go

```diff
@@ -55,10 +57,10 @@ func NewDumper(opts ConnOpts, rdr reader.Reader) (dumper Dumper, err error) {
 	})
 
 	if dumper == nil && err == nil {
-		err = errors.New("no supported driver found")
+		err = fmt.Errorf("no supported driver found for dumper DSN %q", opts.DSN)
```

Comment:
```suggestion
		err = fmt.Errorf("no supported driver found for dumper")
```
Based on https://github.com/hellofresh/klepto/pull/111/files#r402352477